exporter/observiqexporter/config.go

```diff
@@ -60,3 +64,35 @@ func (c *Config) validateConfig() error {
 
 	return nil
 }
+
+// Default agent name will be the hostname
+func defaultAgentName() string {
+	const fallbackAgentName = "otel collector"
+
+	if hn, err := os.Hostname(); err == nil {
+		return hn
+	}
+
+	return fallbackAgentName
+}
+
+// Default agent ID will be UUID based off hostname
+func defaultAgentID() string {
+	const spaceUUIDString = "6dcea175-1870-412d-8a91-2a6f46eec784" // arbitrary space UUID.
```

Comment:
Is this value meaningful / necessary?

Reply:
This is completely arbitrary. It's used as the namespace UUID for a UUID v3. Looking back at it, we could probably just use the Nil UUID here, and it'd be more clear and slightly less arbitrary.

Reply:
Removed this value, just using the Nil UUID instead.